Source/Controllers/DataExport/Exporters/SPSQLExporter.m

```diff
@@ -250,7 +250,7 @@ - (void)exportOperation
     // so to address issue #865, where creating a table with a trigger discards NO_AUTO_VALUE_ON_ZERO,
     // by setting SESSION SQL_MODE to the mode used when the trigger was created then resetting SQL_MODE to @OLD_SQL_MODE
     // we add NO_AUTO_VALUE_ON_ZERO to @OLD_SQL_MODE here, for the export file to properly work.
-    [metaString appendString:@"/*!40101 SET @OLD_SQL_MODE='NO_AUTO_VALUE_ON_ZERO', SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;\n"];
+    [metaString appendString:@"/*!40101 SET @OLD_SQL_MODE='@@SQL_MODE', SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;\n"];
```

Comment:
That docu above says something else, right?

Reply:
I know, but it's wrong. Right now, is replacing SQL_MODE, not adding this value, and after de dump, it's restoring a wrong value, which is worse, causing INSERT using the same connection may fail.

Reply:
I'll leave this up to @jamesstout then to verify and test this.

Reply:
Given the lack of clarity on this and a fairly though-out response from @jamesstout before, I'm gonna close this at this time. It can be reopened if we have a clearer understanding of how future changes here may fix issues without introducing new ones. https://github.com/Sequel-Ace/Sequel-Ace/issues/865#issuecomment-776836401